core/src/types/ops.rs

```diff
@@ -264,6 +265,17 @@ impl OpRead {
     pub fn override_content_disposition(&self) -> Option<&str> {
         self.override_content_disposition.as_deref()
     }
+
+    /// Sets the cache-control header that should be send back by the remote read operation.
+    pub fn with_override_cache_control(mut self, cache_control: impl Into<String>) -> Self {
```

Comment:
Oh, I missed this. Please use `&str` directly instead of introducing generics in non-critical code paths.

`with_override_content_disposition` should also be changed.

Reply:
Done!